.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Plio CI
```

Comment:
you might have to update the Github Workflows and Pre-commit doc file to point to this file right?

Reply:
Done.